.jsdoc.js

```diff
@@ -34,18 +34,19 @@ module.exports = {
   source: {
     excludePattern: '(^|\\/|\\\\)[._]',
     include: [
-      'build/src'
+      'build/src',
+      'build/protos'
     ],
     includePattern: '\\.js$'
   },
   templates: {
-    copyright: 'Copyright 2019 Google, LLC.',
+    copyright: 'Copyright 2020 Google LLC',
     includeDate: false,
     sourceFiles: false,
-    systemName: '@google-cloud/firestore',
+    systemName: 'firestore',
```

Comment:
For this, you need to add `"package-name": "@google-cloud/firestore"` to the generator arguments in `synth.py`.

Reply:
Done (and this fixed the issue here)